### PR TITLE
Activation Account

### PR DESCRIPTION
**Pull request by nguyenthaingoc2002**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/65561)

## WHAT (optional)
- Activation Account

## Evidence (Screenshot or Video)
image depends on both `normal` + `not_normal` items.

### CODE REVIEW

**Comment by nguyenthaingoc2002** [comment]
ready

File: app/helpers/account_activations_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
+end
```
Comment (anlvq-1058) [inline app/helpers/account_activations_helper.rb:2]: file này ko dùng thì xóa đi em

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by nguyenthaingoc2002** [comment]
ready

**Pull request merged**